docs/release-notes.rst

```diff
@@ -10,9 +10,9 @@ Release notes
 itemloaders 1.0.4 (2020-11-12)
 ------------------------------
 
--   When adding a :class:`scrapy.item.Item` object as a value into an
+-   When adding a :class:`scrapy.item.scrapy.Item` object as a value into an
```

Comment:
hm, this path doesn't look right

Reply:
If that's hard to make it work, it's an option to just remove `:class:` here; there won't be a link, but it's not a big deal in release notes.

Reply:
I see that the path works, but I'm not sure why; it looks weird :) Do you have an explanation for it @VMRuiz?

Reply:
Yes, it looks weird but thats how its anchored in the Scrapy docs: 

https://docs.scrapy.org/en/latest/topics/items.html#scrapy.item.scrapy.Item

Reply:
Thanks @VMRuiz for checking it. I suspect after some cleanup these URLs may change again in future, to become more reasonable, but let's solve it separately, if it happens.